docs/documentation/dependent-resources.md

```diff
@@ -169,26 +198,26 @@ instances are managed by JOSDK, an example of which can be seen below:
 ```java
 
 @ControllerConfiguration(
-        labelSelector = SELECTOR,
```

Comment:
I don't get this, we are both using the same formatter from maven, how was this not formatted? It should fail also on build if not properly formatted

Reply:
I don't know but I've seen weird things going on with format… it doesn't appear as deterministic as it should be (or rather, the determinism is not obvious to me) 😓